mne/io/base.py

```diff
@@ -2186,13 +2202,17 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
         write_int(fid, FIFF.FIFF_FIRST_SAMPLE, first_samp)
 
     # previous file name and id
+    if split_naming == 'elekta':
+        data = part_idx - 1
+    else:
+        data = part_idx - 2
```

Comment:
data is not a great name for an "int"

can it be a name based on part_idx ? part_idx_tag ?

Reply:
i got the name data from the helper function ;) but i will use a more descriptive one